### ISSUE TITLE
[Bug] The example YAML is wrong for maria DB

### ISSUE THREAD

**Issue by watchinthewheels20** [rommapp/romm#713]
Hi Guys

Simple documentation error I spotted that's been driving me mad trying to get this installed. Maria DB requires that the environment variables be:

      - MARIADB_ROOT_PASSWORD=
      - MARIADB_DATABASE=
      - MARIADB_USER=
      - MARIADB_PASSWORD=

rather than the MYSQL versions that are still in the documentation, took me a while to work that out, worth updating the example .yaml to reflect the change.

EDIT:I just found that the documentation around default usernames/passwords is also incorrect. The docs say that the default auth is admin/admin it is actually admin and blank password. Also the ROMM_AUTH_USERNAME and ROM_AUTH_PASSWORD variables do not seem to work.

**Comment by gantoine** [comment]
We'll have this fixed in the 3.1 release.

**Comment by gantoine** [comment]
> rather than the MYSQL versions that are still in the documentation

MariaDB [accepts both MARIADB_ and MYSQL_ env variables](https://mariadb.com/kb/en/mariadb-server-docker-official-image-environment-variables/), so either should work!

> default auth is admin/admin it is actually admin and blank password

Code defaults to "admin", it's just your env variable is wrong and should be `ROMM_AUTH_PASSWORD`

```
ROMM_AUTH_PASSWORD: Final = os.environ.get("ROMM_AUTH_PASSWORD", "admin")
```
